packages/graphiql/src/components/ToolbarButton.tsx

```diff
@@ -33,13 +34,14 @@ export class ToolbarButton extends React.Component<
 
   render() {
     const { error } = this.state;
+    const { t } = this.props;
     return (
       <button
         className={'toolbar-button' + (error ? ' error' : '')}
         onClick={this.handleClick}
-        title={error ? error.message : this.props.title}
+        title={t(error ? error.message : this.props.title)}
         aria-invalid={error ? 'true' : 'false'}>
-        {this.props.label}
+        {t(this.props.label)}
```

Comment:
hmm.. not sure if its more efficient to translate it when passing the prop, or at render. what do you think?

Reply:
Yeah, it translates once it receives prop